source/main/gfx/OgreSubsystem.cpp

```diff
@@ -92,6 +92,10 @@ bool OgreSubsystem::Configure()
     miscParams["vsync"] = ropts["VSync"].currentValue;
     miscParams["gamma"] = ropts["sRGB Gamma Conversion"].currentValue;
 #if OGRE_PLATFORM == OGRE_PLATFORM_WIN32
+    const auto rd = ropts["Render Device"];

```

Comment:
It is called "Rendering Device"
```suggestion
    const auto rd = ropts["Rendering Device"];
```